### PR TITLE
Support /credit_card/country input

### PR DESCRIPTION
**Pull request by horgh**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## v2.2.0
+
+* Added the input `/credit_card/country`. This is the country where the
```
Comment (oschwald) [inline CHANGELOG.md:5]: The tense on this is different than the other entries! I don't have a strong opinion on which is "right" as long as the document is consistent.

**Review by oschwald — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (horgh) [reply CHANGELOG.md:5]: Yeah, I was thinking we could switch actually as the other way is different from the other APIs. It'd be simpler if we were more consistent across the APIs. That said I'm ok changing it too.

Comment (oschwald) [reply CHANGELOG.md:5]: Ah, ok. That seems fine.

**Review by oschwald — DISMISSED** [review]

File: spec/components/credit_card_spec.rb [hunk 2]
```diff
@@ -87,5 +87,11 @@
         was_3d_secure_successful: true,
       )
     end
+
+    it 'accepts country code' do
```
Comment (FreeKitty) [inline spec/components/credit_card_spec.rb:91]: suggestion: for the sake of consistency with the other tests, the descriptions could be something along the lines of "does not raise an exception  when country code is given but issuer_id_number" is not provided.

**Review by FreeKitty — APPROVED** [review]

**Pull request merged**